DESIGN.md

```diff
@@ -112,7 +112,7 @@ automatically.
 
 ### Background tasks
 
-The repositories <https://github.com/gitster/git> (and
+The repositories <https://github.com/ttaylorr/git> (and
```

Comment:
GitGitGadget was designed long before you were designated interim maintainer. Therefore, this change is incorrect.